libraries/transformer/src/main/java/androidx/media3/transformer/FrameworkMuxer.java

```diff
@@ -276,9 +283,64 @@ private static ImmutableList<String> getSupportedVideoSampleMimeTypes() {
     if (SDK_INT >= 24) {
       supportedMimeTypes.add(MimeTypes.VIDEO_H265);
     }
+    if (SDK_INT >= 33) {
+      supportedMimeTypes.add(MimeTypes.VIDEO_DOLBY_VISION);
+    }
     if (SDK_INT >= 34) {
       supportedMimeTypes.add(MimeTypes.VIDEO_AV1);
     }
     return supportedMimeTypes.build();
   }
+
+  // Get Dolby Vision profile
+  // Refer to https://professionalsupport.dolby.com/s/article/What-is-Dolby-Vision-Profile
+  @RequiresApi(33)
+  private static int getDvProfile(Format format) {
+    // Currently, only profile 8 is supported for encoding
+    // TODO: set profile ID based on format.
+    return MediaCodecInfo.CodecProfileLevel.DolbyVisionProfileDvheSt;
+  }
+
+  // Get Dolby Vision level
+  // Refer to https://professionalsupport.dolby.com/s/article/What-is-Dolby-Vision-Profile
```

Comment:
Please write it as a java doc comment. Same comment for other method.

Reply:
OK.